src/MagnumPlugins/BasisImageConverter/BasisImageConverter.conf

```diff
@@ -0,0 +1,44 @@
+# [configuration]
+[configuration]
+threads=1
+
+compression_level=1
+
+y_flip=false
+debug=false
+enable_debug_printf=false
+debug_images=false
+global_sel_pal=false
+no_auto_global_sel_pal=false
+perceptual=true
+no_endpoint_rdo=false
+no_selector_rdo=false
+read_source_images=false
+write_output_basis_files=false
+compute_stats=false
+check_for_alpha=true
+force_alpha=false
+multithreading=true
```

Comment:
Leftover? :) It's implicit from the `threads`, right?

Reply:
Yes